### DISCUSSION TITLE
Possibility to disable Tag generation.

### DISCUSSION THREAD

**Discussion by Sebastriani** [open-webui/open-webui#6708] [General]
**Is your feature request related to a problem? Please describe.**
I just realized while auditing chats with Langfuse that the chat is generating a BIG system message to create Tags... At my company we are using Open WebUI with GPT API, so these feature consumes a lot of tokens.

**Describe the solution you'd like**
A simple slide to completelly disable Tag generation would be perfect.

**Describe alternatives you've considered**
I've inserted just a dot (.) in the "Tag Generation Prompt" configuration but still, this sends an additional unnecessary prompt to GPT API

**Additional context**
For simplicity I'm using the Docker Image to deploy the service in Azure so I cannot modify the code myself.

Thanks

**Comment by Sebastriani** [comment]
@tjbck  Why was this converted to a discussion?? I created it as a Feature Request, as a Discussion it will never be properly treated...

**Comment by tjbck** [comment]
Please do not spam. You can disable auto tag generation from Settings > Interface.

![image](https://github.com/user-attachments/assets/fe081513-4e6e-43d5-a8bf-5d72d9344cde)

If you're looking for an option to disable this feature globally, feel free to make a PR.

**Comment by Sebastriani** [comment]
Spam? What do you mean "Spam", I'm being absolutelly serious on the subject, wouldn't it be easier if you answered this solution from the beginning??

EDIT: Also... This solutions is in a per user basis, I need to disable this as Admin for all Users, but ok, I'll stop "Spamming" and try to fix it my self and do a PR.

**Comment by Sebastriani** [comment]
@tjbck  I created a fork and made the changes to be able to globaly disable the Tags Generation.

I don't create a PR because your PR template says that first-time contributors should open a discussion first...

This is the commit in my fork:
https://github.com/Sebastriani/open-webui/commit/e1114bfa4cedb82202697cc6d552cc1c7fb3dd1f

This is the change in the UI
![image](https://github.com/user-attachments/assets/a4b2c642-c3c8-4599-81de-4e1f483faea9)

If the setting is disabled, the backend just responds with a 200 and does not generate the tags.

What else should I do now to be allowed to generate the PR?

**Reply by tjbck** [reply]
Looks great! PR welcome!

**Reply by Sebastriani** [reply]
Done, thanks

**Comment by DReffects** [comment]
has this been added? i am unable to disable title generation globally as well

**Reply by Classic298** [reply]
you can disable it in the admin panel > interface

**Reply by DReffects** [reply]
thanks, i tried that before to no avail but i guess i was to impatient :-) its working now!
